src/services/lib/bitcoingreen-lib.js

```diff
@@ -22,3 +24,14 @@ export const importAddressFromPrivateKey = privateKey => {
   const { address } = bitcoin.payments.p2pkh({ pubkey: keyPair.publicKey });
   return address;
 };
+
+export const setTransaction = (txUtxos, txUtxoValue, amount, receiveAddress, senderAddress) => {
+  const rawTransaction = new bitcoin.TransactionBuilder();
+  for (let i = 0; i < txUtxos.length; i += 1) {
+    rawTransaction.addInput(txUtxos[i].txId, txUtxos[i].vout);
+  }
+
+  const change = txUtxoValue - amount - config.FEE_AMOUNT;
```

Comment:
FEE_AMOUNT is defined?

Reply:
following by issue25: fee - this amount is fixed to a constant FEE_AMOUNT that we define in the app

Reply:
good